package.json

```diff
@@ -25,7 +25,7 @@
 	},
 	"devDependencies": {
 		"@types/estree": "0.0.42",
```

Comment:
`@types/estree` is needed for the type definitions as part of the public interface, so it belongs in `dependencies`, not `devDependencies`. Also I suggest to just update this to `^1.0.0` while you’re at it.

Reply:
Good idea, updated

Reply:
this also resolves https://github.com/Rich-Harris/estree-walker/issues/35